src/PythonMigrationViewExtension/PythonMigrationViewExtension.cs

```diff
@@ -120,7 +121,7 @@ private void OnMigrationAssistantRequested(object sender, EventArgs e)
             var parentWindow = Window.GetWindow(btn);
 
             var node = sender as PythonNode;
-            var viewModel = new PythonMigrationAssistantViewModel(node);
+            var viewModel = new PythonMigrationAssistantViewModel(node, DynamoViewModel.CurrentSpace, DynamoViewModel.Model.PathManager);
```

Comment:
@radumg you should be able to get both the current workspace and the path manager from the LoadedParams without referencing the DynamoViewModel.